Tools/Scripts/webkitpy/style/checker.py

```diff
@@ -326,6 +326,10 @@
       os.path.join('Source', 'WebCore', 'PAL', 'pal', 'spi')],
      ["-readability/naming/underscores"]),
 
+    ([  # simd.h structs all have underscores in them.
+     os.path.join('Source', 'WebCore', 'platform', 'graphics', 'cocoa', 'TransformationMatrixCocoa.cpp')],
```

Comment:
This is a bit excessive and specific.  Can't we just ignore the checker for the probably only 1 PR that will have this as an issue?

Reply:
I believe we can! Will take this diff out